docker-compose/docker-compose.yml

```diff
@@ -51,7 +51,7 @@ services:
 
   # WHISK CONTROLLER
   controller:
-    image: whisk/controller:latest
+    image: openwhisk/controller:6a56c45
```

Comment:
+1 to the `latest` version too.

I can put in a jenkinsfile for verifying docker-compose. Where do you put the jenkins job for building and pushing images?